pkg/worker/worker.go

```diff
@@ -36,6 +36,7 @@ import (
 func GatherResults(waitfile string, url string) error {
 	var inputFileName []byte
 	var err error
+	var outfile * os.File 
```

Comment:
gofmt this bad boy

Reply:
It does, on save.. 

    "go.formatOnSave": true, 
    "go.formatTool": "gofmt",

Reply:
weird!! When I save it it moves that * over to be `var outfile *os.File`

I literally don't care on this issue. Feel free to push.